Dockerfile

```diff
@@ -1,14 +1,14 @@
-FROM node:14-alpine as build
+FROM node:16-alpine as build
 
 RUN mkdir /app
 WORKDIR /app
 COPY . /app
-RUN apk add --update python gcc g++ make && \
+RUN apk add --update gcc g++ make && \
```

Comment:
Not sure why python won't install in 16, but the image was built successfully without it.